packages/markdown-docx/src/CiceroMarkToOOXML/index.js

```diff
@@ -60,6 +60,28 @@ class CiceroMarkToOOXMLTransfomer {
      * @returns {string} OOXML for the given node
      */
     getNodes(node, counter, parent = null) {
+        if (this.getClass(node) === definedNodes.variable) {
+            const tag = node.name;
+            const type = node.elementType;
+            if (Object.prototype.hasOwnProperty.call(counter, tag)) {
+                counter = {
+                    ...counter,
+                    [tag]: {
+                        ...counter[tag],
+                        count: ++counter[tag].count,
+                    },
+                };
```

Comment:
No change here. But I just wanted to confirm in you know why `counter` variable is introduced in the first place.

Reply:
@algomaster99 
A variable of the same type, say `Shipper` can have multiple instances in a template. The `counter` variable counts the number of instances of that variable. Now using this counter, a title is generated for the variable content control to distinguish it, This is my understanding of the above.

Reply:
> Now using this counter, a title is generated for the variable content control to distinguish it

Almost got it. A more precise answer - each content control in the document must have a unique title.